composer.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "qa-tools/behat-example",
+	"license": "BSD-3-Clause",
+	"require": {
+		"php": ">=5.3.1",
+		"behat/mink-selenium2-driver": "~1.2.0",
```

Comment:
We need to explicitly specify Mink dependency here as well. Yes I know that it will be indirectly detected through driver dependencies anyway.